zipline/lib/adjusted_array.py

```diff
@@ -237,6 +238,25 @@ def inspect(self):
             adjustments=self.adjustments,
         )
 
+    def map_labels(self, func):
```

Comment:
I might call this something like `update_labels` to imply that this operates in place.

Reply:
Ah nice, I'll update